extensions/amp-script/0.1/test-e2e/test-amp-script.js

```diff
@@ -73,5 +71,27 @@ describes.endtoend(
 
       await expect(realRatio).to.equal(targetRatio);
     });
+
+    // In layout=container, amp-script requires mutations to be backed by
+    // user gestures. This ensures that this requirement is also enforced
+    // on load AKA "hydration".
+    it('should not mutate on load in layout=container', async () => {
```

Comment:
Is there anywhere where we are testing the inverse? e.g. `it('should mutate on load in layout=fixed')`.

Reply:
Yea, the first two tests cover that case.